vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -0,0 +1,165 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import javax.transaction.Transactional;
+
+import org.apache.tika.Tika;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.data.ImageContentBlockRepository;
+import edu.asu.diging.vspace.core.data.ImageRepository;
+import edu.asu.diging.vspace.core.data.SlideRepository;
+import edu.asu.diging.vspace.core.data.TextContentBlockRepository;
+import edu.asu.diging.vspace.core.exception.FileStorageException;
+import edu.asu.diging.vspace.core.factory.IImageBlockFactory;
+import edu.asu.diging.vspace.core.factory.IImageFactory;
+import edu.asu.diging.vspace.core.factory.ITextBlockFactory;
+import edu.asu.diging.vspace.core.file.IStorageEngine;
+import edu.asu.diging.vspace.core.model.IContentBlock;
+import edu.asu.diging.vspace.core.model.IImageBlock;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.IVSImage;
+import edu.asu.diging.vspace.core.model.impl.ImageBlock;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+import edu.asu.diging.vspace.core.model.impl.TextBlock;
+import edu.asu.diging.vspace.core.model.impl.VSImage;
+import edu.asu.diging.vspace.core.services.IContentBlockManager;
+
+@Transactional
+@Service
+public class ContentBlockManager implements IContentBlockManager {
+
+    @Autowired
+    private ITextBlockFactory textBlockFactory;
+
+    @Autowired
+    private IImageBlockFactory imageBlockFactory;
+
+    @Autowired
+    private SlideManager slideManager;
+
+    @Autowired
+    private TextContentBlockRepository textBlockRepo;
+
+    @Autowired
+    private ImageContentBlockRepository imageBlockRepo;
+
+    @Autowired
+    private IImageFactory imageFactory;
+
+    @Autowired
+    private ImageRepository imageRepo;
+
+    @Autowired
+    private SlideRepository slideRepo;
+
+    @Autowired
+    private IStorageEngine storage;
+
+    private int BlockInOrder = 0;
```

Comment:
since when do we capitalize variables??? Also, a global variable? no, no, no, no, no, no, no,....